app/services/submissions/update_service.rb

```diff
@@ -5,7 +5,7 @@ def call(submission, params, role)
         submission.with_lock do
           submission.current_version += 1
           EventAdditionService.call(submission, params)
-          submission.update!(params.permit(:application_state, :application_risk))
+          submission.update!(params.permit(:application_risk).merge(state: params[:application_state]))
```

Comment:
Should this not be `state` too?

Reply:
Similar thing - the HTTP API hasn't changed, so the param name will still be `application_state`